js/resume.min.js

```diff
@@ -1 +1,40 @@
-!function(e){"use strict";e('a.js-scroll-trigger[href*="#"]:not([href="#"])').click(function(){if(location.pathname.replace(/^\//,"")==this.pathname.replace(/^\//,"")&&location.hostname==this.hostname){var t=e(this.hash);if((t=t.length?t:e("[name="+this.hash.slice(1)+"]")).length)return e("html, body").animate({scrollTop:t.offset().top},1e3,"easeInOutExpo"),!1}}),e(".js-scroll-trigger").click(function(){e(".navbar-collapse").collapse("hide")}),e("body").scrollspy({target:"#sideNav"})}(jQuery);
\ No newline at end of file
+!(function(e) {
+  "use strict";
+  e('a.js-scroll-trigger[href*="#"]:not([href="#"])').click(function() {
+    if (
+      location.pathname.replace(/^\//, "") ==
+        this.pathname.replace(/^\//, "") &&
+      location.hostname == this.hostname
+    ) {
+      var t = e(this.hash);
+      if ((t = t.length ? t : e("[name=" + this.hash.slice(1) + "]")).length)
+        return (
+          e("html, body").animate(
+            { scrollTop: t.offset().top },
+            1e3,
+            "easeInOutExpo"
+          ),
+          !1
+        );
+    }
+  }),
+    e(".js-scroll-trigger").click(function() {
+      e(".navbar-collapse").collapse("hide");
+    }),
+    e("body").scrollspy({ target: "#sideNav" });
+})(jQuery);
+var fisrtName = document.querySelector(".mb-0");
```

Comment:
Using queries like this is very fragile. What if someone changes the class in the HTML to `mb-1` for example? Or adds another element using this margin style before the one you want? You should use separate identifiers in the HTML for your queries whenever possible. In the case of hard-coded values like this though, they should go directly in the HTML rather than be set using Javascript (it's OK to do this for practice, but it would be a bad idea for a real site).